tests/system/Session/SessionTest.php

```diff
@@ -248,7 +248,7 @@ public function testIssetReturnsTrueOnSuccess()
 
         $_SESSION['foo'] = 'bar';
 
-        $this->assertTrue(isset($session->foo));
+        $this->assertObjectHasAttribute('foo', $session);
```

Comment:
This assertion fails.

Reply:
You can register `Rector\PHPUnit\Rector\MethodCall\AssertIssetToSpecificMethodRector` into Option::SKIP

```php
use Rector\PHPUnit\Rector\MethodCall\AssertIssetToSpecificMethodRector;

    $parameters->set(Option::SKIP, [
             // ....

            AssertIssetToSpecificMethodRector::class => [
                     __DIR__ . '/tests/system/Entity/EntityTest.php',
                    __DIR__ . '/tests/system/Session/SessionTest.php',
           ],
    ]);
```

and then rollback change for `tests/system/Entity/EntityTest.php`

Reply:
If `AssertIssetToSpecificMethodRector` cause invalid result in many places, we can just skip it al together:

```php
use Rector\PHPUnit\Rector\MethodCall\AssertIssetToSpecificMethodRector;

    $parameters->set(Option::SKIP, [
             // ....

            AssertIssetToSpecificMethodRector::class ;
    ]);
```

Reply:
I would rather change the test code to remove the skip `AssertIssetToSpecificMethodRector`.

Thank you for telling me about rector!